### PR TITLE
feat(keyboard): Add keyboard effects

### PR DESCRIPTION
**Pull request by alexander-heimbuch**
- Left, Right: Scrub forward/backward
- Space: Play/Pause
- Use Github Pages for Docs
- Use Yarn

### CODE REVIEW

File: src/store/effects/keyboard.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  store.dispatch(actions.updatePlaytime(time))
+}
+
+const scrubBackward = store => () => {
+  const state = store.getState()
+  const playtime = get(state, 'playtime')
+  const duration = get(state, 'duration')
+  let time = playtime - Math.pow(2, calcModifier())
+
+  time = time > duration ? duration : time
```
Comment (mplabs) [inline src/store/effects/keyboard.js]: I think you need to check wether you scrub below the beginning of the track, here...

File: src/store/effects/keyboard.test.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  t.is(keyEventHandler.getCall(2).args[0], 'space')
+})
+
+test(`keyboardEffect: it registers chapter modifiers`, t => {
+  effect()(store)
+  t.is(keyEventHandler.getCall(3).args[0], 'alt+right')
+  t.is(keyEventHandler.getCall(4).args[0], 'alt+left')
+})
+
+test(`keyboardEffect: scrub forward dispatches UPDATE_PLAYTIME`, t => {
```
Comment (mplabs) [inline src/store/effects/keyboard.test.js]: Maybe add test to not update playtime beyond track bounds here...

File: src/store/effects/keyboard.test.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  effect('right')(store)
+  const action = store.dispatch.firstCall.args[0]
+  t.is(action.type, 'UPDATE_PLAYTIME')
+  t.truthy(action.payload > 10)
+})
+
+test(`keyboardEffect: scrub backward dispatches UPDATE_PLAYTIME`, t => {
```
Comment (mplabs) [inline src/store/effects/keyboard.test.js]: Maybe add test to not update playtime below track bounds here...

File: src/store/effects/keyboard.test.js (hunk 2 above)
Comment (alexander-heimbuch) [inline src/store/effects/keyboard.test.js]: fixed it :)

**Pull request merged**